src/Dictionaries/DictionaryHelpers.h

```diff
@@ -122,6 +122,47 @@ class DictionaryStorageFetchRequest
         }
     }
 
+    DictionaryStorageFetchRequest(const DictionaryStructure & structure,
```

Comment:
Can we reduce code duplication from the constructor above? They are almost the same

Reply:
done via std::optional